public/JSONtoHTML.coffee

```diff
@@ -0,0 +1,211 @@
+# Variables
```

Comment:
Keep all lines of max 80 chars long please.

Reply:
ok, but it doesn,t make a huge diff?

Reply:
We just keep max line length of 80 chars in all of our files, so we have to do it here too.